clients/src/test/java/org/apache/kafka/common/record/FileRecordsTest.java

```diff
@@ -316,7 +316,7 @@ public void testConvertNonCompressedToMagic1() throws IOException {
         try (FileRecords fileRecords = FileRecords.open(tempFile())) {
             fileRecords.append(records);
             fileRecords.flush();
-            Records convertedRecords = fileRecords.toMessageFormat(Record.MAGIC_VALUE_V1);
+            Records convertedRecords = fileRecords.toMessageFormat(Record.MAGIC_VALUE_V1, TimestampType.CREATE_TIME);
```

Comment:
No particular reason. We have to choose one of them when up-converting. This just verifies that it uses what we set.